src/main/java/org/apache/maven/report/projectinfo/TeamReport.java

```diff
@@ -83,9 +116,22 @@ public boolean canGenerateReport() throws MavenReportException {
     }
 
     @Override
-    public void executeReport(Locale locale) {
-        ProjectTeamRenderer r =
-                new ProjectTeamRenderer(getSink(), project.getModel(), getI18N(locale), locale, showAvatarImages);
+    public void executeReport(Locale locale) throws MavenReportException {
+        AvatarsProvider avatarsProvider = avatarsProviders.get(avatarProviderName);
+        if (avatarsProvider == null) {
+            throw new MavenReportException("No AvatarsProvider found for name " + avatarProviderName);
```

Comment:
Would it make sense to instead simply not show avatars in the report?

Reply:
It is two different case wrong configuration - not existing provider and not available image server.

In both case we should break a build to prevent of unwanted result